### ISSUE TITLE
SpaCy Tutorial 5: Labeling the validation set

### ISSUE THREAD

**Issue by scpkobayashi** [koaning/spacy-youtube-material#3]
Hi Vincent,
Thanks a lot for your spaCy videos, I learned a lot from them.
Actually, I have a question about the validation set you used for the evaluation in part 5 of the tutorial.
How did you label your validation set knowing that we can't use either the rule-based model or the statistical model to do the job?
The only solution that came to my mind was to build another rule-based model based on the test set with extra effort to match every single exception.  
But then do we need to build a new model for every new test set?

**Comment by koaning** [comment]
I'm not sure I understand the question. But I'll try to explain the logic I had in my mind. 

I used a rule based model to gather examples. This made labelling a whole lot quicker because I was able to say "yep" or "nope". I did not need to read through all the examples to add a programming language. 

This does mean that the examples that I have are somewhat biased. This bias propogates in the training set, validation set and the statistical model. So I figured I would compare the results of both models and try to learn from it. Each time I iterate on both models but also my data. By repeating this over and over I figured I'd get to a place I want to be. 

It should be said though, I am currently waiting for spaCy 3.0 before continueing with this work. The issue with the videos sofar is that I'm not really working towards a proper problem yet and therefore I don't really have a proper validation set. This will get adressed in the next video.
